### ISSUE TITLE
Failed to convert quantized segmentation model from segmentation_models.pytorch

### ISSUE THREAD

**Issue by hgaiser** [google-ai-edge/ai-edge-torch#381]
### Description of the bug:

I'm trying to convert the following (quantized) model:

```python
# Disable GPU for model conversion to tflite.
# Fix for https://github.com/google-ai-edge/ai-edge-torch/issues/326
# This may be removed when the issue is resolved.
# Without this fix, the training crashes with `Unknown PJRT_DEVICE 'CUDA'`.
import os
os.environ['PJRT_DEVICE'] = 'CPU'

import segmentation_models_pytorch as smp
import torch
import ai_edge_torch
from torch.ao.quantization.quantize_pt2e import convert_pt2e, prepare_qat_pt2e
from torch.ao.quantization.quantizer.xnnpack_quantizer import XNNPACKQuantizer, get_symmetric_quantization_config

model = smp.FPN('resnet34')

example_inputs = (torch.randn(2, 3, 512, 512),)
model = torch.export.export_for_training(model, example_inputs).module()
quantizer = XNNPACKQuantizer().set_global(get_symmetric_quantization_config())
model = prepare_qat_pt2e(model, quantizer)

model = convert_pt2e(model)
torch.ao.quantization.move_exported_model_to_eval(model)
edge_model = ai_edge_torch.convert(model, example_inputs)
```

But this fails to complete.

### Actual vs expected behavior:

I expect a converted model, instead I get the following error:

> loc("<XLA_MLIR_DEBUGINFO_BEGIN>torch.fx.graph_module.GraphModule.__new__.<locals>.GraphModuleImpl;<XLA_MLIR_DEBUGINFO_END>xla__dequantize_tensor"): error: 'mhlo.uniform_dequantize' op operand #0 must be ranked tensor of 2/4/8/16/32-bit uniform quantized signed integer or 2/4/8/16/32-bit uniform quantized unsigned integer or 2/4/8/16/32-bit uniform quantized per axis signed integer or 2/4/8/16/32-bit uniform quantized per axis unsigned integer values, but got 'tensor<64x3x7x7xi8>'

(full output is in the attachment)

[output.txt](https://github.com/user-attachments/files/17902291/output.txt)

### Any other information you'd like to share?

Attached is the requirements.txt of the installed packages in my virtualenv.

[requirements.txt](https://github.com/user-attachments/files/17902308/requirements.txt)

**Comment by pkgoogle** [comment]
With the latest code and commit `e3d0d3`, my process gets killed. It appears this is bugged either way.

**Comment by hgaiser** [comment]
> With the latest code and commit `e3d0d3`, my process gets killed. It appears this is bugged either way.

Ah that's not good. Does it finish with an error if you use `ai-edge-torch-nightly`?

**Comment by pkgoogle** [comment]
Nightly, also kills my process.

**Comment by hgaiser** [comment]
I'm going to look into this more today, will update with my findings. So far I've noticed that even a simple (non-quantized) example fails with `ai-edge-torch-nightly` with a `Segmentation fault`:

```py
import os
os.environ['PJRT_DEVICE'] = 'CPU'

import ai_edge_torch
import torch
import torchvision

torch_model = torchvision.models.resnet18(torchvision.models.ResNet18_Weights.IMAGENET1K_V1)
sample_args = (torch.randn(1, 3, 224, 224),)

edge_model = ai_edge_torch.convert(torch_model.eval(), sample_args)
```

Running that same script with `ai-edge-torch` instead does succeed. This seems like an unrelated issue, but worth mentioning.

**Comment by hgaiser** [comment]
I managed to reproduce this issue also with `torchvision`, so I'm closing this issue in favor of https://github.com/google-ai-edge/ai-edge-torch/issues/381.
